index.bs

```diff
@@ -82,7 +91,29 @@ decoder part of a codec. The processed media can be consumed by any destination
 that can take a MediaStreamTrack, including HTML &lt;video&gt; and &lt;audio&gt; tags,
 RTCPeerConnection, canvas or MediaRecorder.
 
-# Terminology # {#terminology}
+This specification explicitly aims to support the following use cases:
+- *Video processing*: This is the "Funny Hats" use case, where the input is a single video track and the output is a transformed video track.
+- *Audio processing*: This is the equivalent of the video processing use case, but for audio tracks. This use case overlaps partially with the {{AudioWorklet}} interface, but the model provided by this specification differs in significant ways:
+    - Pull-based programming model, as opposed to {{AudioWorklet}}'s clock-based model. This means that processing of each single block of audio data does not have a set time budget.
+    - Offers direct access to the data and metadata from the original {{MediaStreamTrack}}. In particular, timestamps come directly from the track as opposed to an {{AudioContext}}.
+    - Easier integration with video processing by providing the same API and programming model and allowing both to run on the same scope.
+    - Does not run on a real-time thread. This means that the model is not suitable for applications with strong low-latency requirements.
+
+    These differences make the model provided by this specification more
+    suitable than {{AudioWorklet}} for processing that requires more tolerance
+    to transient CPU spikes, better integration with video
+    {{MediaStreamTrack}}s, access to track metadata (e.g., timestamps), but
+    not strong low-latency requirements such as local audio rendering.
+
+    An example of this would be <a href="https://arxiv.org/abs/1804.03619">
+    audio-visual speech separation</a>, which can be used to combine the video
+    and audio tracks from a speaker on the sender side of a video call and
+    remove noise not coming from the speaker (i.e., the "Noisy cafeteria" case).
+    Other examples that do not require integration with video but can benefit
+    from the model include echo detection and other forms of ML-based noise
+    cancellation.
+  - *Multi-source processing*: In this use case, two or more tracks are combined into one. For example, a presentation containing a live wheather map and a camera track with speaker can be combined to produce a weather report application. Audio-visual speech separation is another case of multi-source processing.
```

Comment:
I don't know if it's necessary to provide additional examples, but here are a couple more where multiple streams is important:
 - Audio Panning: mixing several mono audio tracks into a combined stereo audio track such that the listener can use sound localization to differentiate the source tracks.
 - Roundtable: rendering multiple remote video tracks into a single video track (or into a single canvas)

It may be that AudioWorklet can accomplish the Audio Panning use case -- I haven't looked into it. It's probably possible to handle the Roundtable use case by rendering <video> elements and using requestVideoFrameCallback.

Reply:
Landing without the extra examples, but might add them in a follow-up PR once I understand them better.